tests/Custom/TestCase.php

```diff
@@ -37,4 +37,16 @@ protected function getConfig(): WebdriverClassicConfig
     {
         return WebdriverClassicConfig::getInstance();
     }
+
+    /**
+     * @before
+     *
+     * @return void
+     */
+    protected function checkSkippedTest()
+    {
+        if (null !== $message = self::getConfig()->skipMessage(get_class($this), $this->getName(false))) {
```

Comment:
I would enclose the assignment in brackets (or put it in a separate line), it would be more readable.
I would also replace `get_class($this)` with `static::class`:
```php
$message = $this->getConfig()->skipMessage(self::class, $this->getName(false));

if ($message !== null) {
    $this->markTestSkipped($message);
}
```

Reply:
Done.